<?xml version="1.0"?>
<launch>
     


     <arg name="dbg_args"   default="gdb -ex run --args"/>
     <arg name="debug_args"   default=""/>

<node pkg="pspnet_ros" type="pclpub_node" name="pclpub_node" output="screen">

	  
         
          <param name="cost_image_topic"        value="pspnet_output/cost_image"/> 
          <param name="sync_depth_image_topic"        value="pspnet_output/sync_depth_raw"/> 
          <param name="depth_cam_info_topic"        value="camera/depth/camera_info"/>
          <param name="point_cloud_frame"        value="depth_camera_optical_frame"/>  
          <param name="tf_prefix"        value=""/>
        
</node>
<!--
<node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 -1.57 0 -1.57  map camera_optical_frame  10" /> -->

</launch>
